services/shhext/chat/encryption.go

```diff
@@ -130,7 +130,15 @@ func (s *EncryptionService) ProcessPublicBundle(b *Bundle) error {
 }
 
 // DecryptPayload decrypts the payload of a DirectMessageProtocol, given an identity private key and the sender's public key
-func (s *EncryptionService) DecryptPayload(myIdentityKey *ecdsa.PrivateKey, theirIdentityKey *ecdsa.PublicKey, msg *DirectMessageProtocol) ([]byte, error) {
+func (s *EncryptionService) DecryptPayload(myIdentityKey *ecdsa.PrivateKey, theirIdentityKey *ecdsa.PublicKey, msgs *map[string]*DirectMessageProtocol) ([]byte, error) {
```

Comment:
Maybe we can create a name type that equates to `string` but makes it easier to see what the map contains? E.g. `type InstallationID string`